src/tracked_struct.rs

```diff
@@ -742,9 +756,10 @@ where
         revision: Revision,
     ) -> MaybeChangedAfter {
         let zalsa = db.zalsa();
-        let data = Self::data(zalsa.table(), input);
+        let data = Self::data_raw(zalsa.table(), input);
+        let created_at = unsafe { (*data).created_at };
```

Comment:
I wonder if `created_at` should be an `AtomicRevision` because we mutate it in place during `update`